examples/python/esx-mtu-fixer/README.md

```diff
@@ -0,0 +1,37 @@
+# ESX Maximum transmission unit fixer
+
+This is a remidiation function used which will be triggered when a VM is powered on. It will make sure that the Maximum transmission unit of the VM Kernel Adapter on all ESX hosts is at least `1500`. You can find out more about why `1500` is an optimal value in the [wikipedia page](https://en.wikipedia.org/wiki/Maximum_transmission_unit).
+
+## Set up
+
+Prerequisites are :
+* Binary [faas-cli](https://docs.openfaas.com/cli/install/)
+* [VEBA](https://flings.vmware.com/vcenter-event-broker-appliance?download_url=https%3A%2F%2Fdownload3.vmware.com%2Fsoftware%2Fvmw-tools%2Fveba%2FvCenter_Event_Broker_Appliance_0.1.0.ova#summary) or [OpenFaaS](https://docs.openfaas.com/deployment/kubernetes/) and [vcenter-connector](https://github.com/openfaas-incubator/vcenter-connector) on top of [kubernetes](https://kubernetes.io/docs/setup/learning-environment/minikube/). It is preferred to go with VEBA as it is single installation.
+* Deployed arbitrary VM on your vCenter to trigger the function when the VM is powered on.
+
+The function needs credentials and endpoint of the vCenter with which the function will interact. You can see how to create a secret containing those credentials in your kubernetes cluster in the [create_secret](./create_secret.sh) script. Your `kubectl` must be configured to communicate with your remote cluster first.
+
+Prepend to the the `image` field `esx-mtu-fixer:1.0.0` in [stack.yml](./stack.yml) your own registry location and tags.
+> Example: If you dockerhub username is `lovedocker` the image name should be `lovedocker/esx-mtu-fixer:1.0.0`
+
+Run `faas up --gateway=<OF_GATEWAY_ENDPOINT>`.
```

Comment:
For new users, this will error out because login is required (VEBA uses basic_auth) and since users have to build the image the template is required (pull first).

Reply:
Also, the `faas` commands might require `--tls-no-verify`

Reply:
You are right I missed those details will fix them ASAP 👍

Reply:
Done ✔️